### ISSUE TITLE
Ecosystem Windows CI

### ISSUE THREAD

**Issue by aloubyansky** [quarkusio/quarkus#9445]
That's a task and a bug. The task is to add Windows to the CI.
The bug is it will most probably fail today according to https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/Adding.20extension.20to.20quarkus-platform.

**Comment by geoand** [comment]
cc @maxandersen

**Comment by geoand** [comment]
I'm not really sure how we can do this effectively given that the job definition is not in our hands

**Comment by maxandersen** [comment]
update our example job definition and we ask by opening issue on the projects to add the new section ?

**Comment by geoand** [comment]
Yeah, that's probably the only why. But then we now have a problem with reporting, since the reporting code will run separately once for Windows and once for Linux.
Should we have separate Quarkus issues for Windows errors?

**Comment by geoand** [comment]
Are we OK with having a separate issue to report Windows runs?

**Comment by aloubyansky** [comment]
I am!

**Comment by geoand** [comment]
@maxandersen ?

**Comment by maxandersen** [comment]
Does it matter ? Aren't afailure a failure no matter the OS ?

I'm +0 on it but if you guys see value in having separate issue pr. OS I'm fine with it.

**Comment by geoand** [comment]
The with the same issue is that it you can have the following scenario:

* Issue gets opened by Windows run
* Issue gets closed by Linux run

That's what I want to avoid with multiple issues. But if you have any better ideas, I'm all ears :(

**Comment by maxandersen** [comment]
ah - I kinda thought they would be opened/closed at the end of the runs - like two parallel jobs and if one fails report failure only fix when all is good.

if that is not feasible then +1 on separate issues.

**Comment by gastaldi** [comment]
This issue should probably be moved to quarkusio/quarkus-ecosystem-ci, as it appears to belong there?
